### DISCUSSION TITLE
Drag&Drop node A on node B to create the edge AB

### DISCUSSION THREAD

**Discussion by MaurizioCasciano** [DerYeger/d3-graph-controller#44] [Ideas]
Hi, is it possible to configure Drag&Drop of nodes in order to create an edge between them?
E.g.: I want to be able to create an edge AB by dragging node A and dropping it on node B.

**Comment by DerYeger** [comment]
Hello, this isn't possible without a lot of customization using [the callbacks exposed in the config](https://graph-controller.yeger.eu/config/#modifiers). 
However, this can be quite tedious.

I chose to not implement that functionality in this library, because it would break touch controls.
I did implement link creatin in [this](https://github.com/DerYeger/apollo-frontend/blob/master/src/app/components/graph/graph.component.ts) project ([demo](https://apollo.yeger.eu/model-checker?graphSource=graphCache&graphKey=Example%20Graph)).

Because of the limitations of touch input I had to decide between supporting node dragging and link creation for mobile screens.
